ConnectorTeklaStructures/ConnectorTeklaStructuresShared/StreamStateManager/StreamStateManager.cs

```diff
@@ -41,31 +43,66 @@ public static void WriteStreamStateList(Model model, List<StreamState> streamSta
       GetOrCreateSpeckleFilePath(model);
     }
 
+    if (_speckleFilePath == null)
+      return;
```

Comment:
done. damn i missed that one

Reply:
Looking at this again, it would be clearer if the 2nd check occurs within the first if block. It initially looked like a mistake in the code, but I see GetOrCreateSpeckleFilePath() tries to create the paht also